modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -416,12 +416,12 @@ spec:
       [sysfs]
       /sys/devices/system/cpu/intel_pstate/max_perf_pct = <x> <1>
 ----
-<1> The `max_perf_pct` controls the maximum frequency the `cpufreq` driver is allowed to set as a percentage of the maximum supported cpu frequency. This value applies to all CPUs. You can check the maximum supported frequency in `/sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq`.
+<1> The `max_perf_pct` controls the maximum frequency the `cpufreq` driver is allowed to set as a percentage of the maximum supported cpu frequency. This value applies to all CPUs. You can check the maximum supported frequency in `/sys/devices/system/cpu/cpu0/cpufreq/cpuinfo_max_freq`. As a starting point, you can use a percentage that caps all CPUs at the `All Cores Turbo` frequency. This is the frequency that all cores will run at when P-states are disabled.
```

Comment:
Reword last sentence to:
This is the frequency that all cores will run at when they are all fully occupied.